src/styles/routingTransitions.ts

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/core'
+import { transitions, sizes } from '@/shared/theme'
+
+export const routingTransitions = css`
+  .${transitions.names.fadeAndSlide}-enter {
+    opacity: 0;
+
+    & .${transitions.names.slide} {
+      transform: translateY(${sizes(8)});
+    }
+  }
+
+  .${transitions.names.fadeAndSlide}-enter-active {
+    opacity: 1;
+    & .${transitions.names.slide} {
+      transform: translateY(0);
+      transition: ${transitions.timings.loading} ${transitions.easing};
```

Comment:
Let's actually add a new timing specific for routing. `transitions.timings.routing` let's say, and let's try `300ms`. It should be used in this file and also in `timeout` passed to `CSSTransition`